### ISSUE TITLE
Files don't always open in the window to the right

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ijt/emacs-nav#95]
```
What steps will reproduce the problem?
1. C-x 2 to split 
2. M-x nav 
3. Move the cursor over a filename and hit enter.

What is the expected output? What do you see instead?
The file should open in the window to the right of the nav window.
Instead it opens in the window below.

 This issue has been observed in emacs 24 and may be present in other versions.
```

Original issue reported on code.google.com by `issac.tr...@gmail.com` on 7 May 2012 at 6:01

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 15ec143d06f7
```

Original comment by `issac.tr...@gmail.com` on 7 May 2012 at 6:06
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in change 879e80b02584.
```

Original comment by `issac.tr...@gmail.com` on 7 May 2012 at 7:08
